OpenEdXMobile/src/main/java/org/edx/mobile/module/download/IDownloadManager.java

```diff
@@ -33,8 +33,8 @@
      * @param dmid
      * @return
      */
-    boolean removeDownload(long dmid);
-    
+    boolean removeDownload(long... dmid);
```

Comment:
Since we are now accepting array of `longs`, we should rename this function to `removeDownloads` and update the javadoc as well.